.tekton/main-pull-request.yaml

```diff
@@ -98,446 +98,8 @@ spec:
   timeouts:
     pipeline: 1h30m0s
 
```

Comment:
[not about this line]
Please add `spec.params` for `clone-depth` and `clone-fetch-tags` for explicitness and to have `main` pipelines similar to others.